client/src/modules/TeamDistribution/components/TeamDistributionCard/CardTitle.tsx

```diff
@@ -0,0 +1,52 @@
+import { Col, Row, Space, Tag, Typography } from 'antd';
+import { TeamOutlined, ClockCircleOutlined } from '@ant-design/icons';
+import { TeamDistributionDto, TeamDistributionDtoRegistrationStatusEnum } from 'api';
+import { DistributionPeriod } from './DistributionPeriod';
+
+type Props = {
+  distribution: TeamDistributionDto;
+};
+
+const { Text, Title } = Typography;
+
+export function CardTitle({ distribution }: Props) {
+  const renderTag = () => {
+    switch (distribution.registrationStatus) {
+      case TeamDistributionDtoRegistrationStatusEnum.Distributed:
+        return (
+          <Tag icon={<ClockCircleOutlined />} color="green">
+            distributed
+          </Tag>
+        );
+      case TeamDistributionDtoRegistrationStatusEnum.Completed:
+        return <Tag icon={<ClockCircleOutlined />}>without team</Tag>;
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <Row gutter={24} wrap justify="space-between">
+      <Col>
+        <Title level={5}>{distribution.name}</Title>
+      </Col>
+      <Col>
+        <Space size="middle" wrap>
+          {distribution.registrationStatus === TeamDistributionDtoRegistrationStatusEnum.Completed ||
+          distribution.registrationStatus === TeamDistributionDtoRegistrationStatusEnum.Distributed
+            ? renderTag()
+            : distribution.minTotalScore !== 0 && (
+                <Text style={{ fontSize: 14 }} type="secondary">
+                  Min score {distribution.minTotalScore}
+                </Text>
+              )}
```

Comment:
seams like you check `distribution.registrationStatus` against `TeamDistributionDtoRegistrationStatusEnum` twice: inside Space element and renderTag function. Unless I'm missing something, you could just move min score element inside `renderTag` function(rename it) and replace that ternary operator with function call. 
```jsx
default:
return distribution.minTotalScore !== 0 
? (
  <Text style={{ fontSize: 14 }} type="secondary">
    Min score {distribution.minTotalScore}
  </Text>) 
: null
```

Reply:
I think it's not the best idea, because we either render the student status in the tag or show the minimum score for registration.

Reply:
never mind then)

Reply:
how about 
```jsx
{renderTag() ?? distribution.minTotalScore !== 0 && (
  <Text style={{ fontSize: 14 }} type="secondary">
    Min score {distribution.minTotalScore}
  </Text>
)}
```